requirements.yml

```diff
@@ -8,3 +8,5 @@
 - src: https://github.com/redhat-nfvpe/auto-kube-dev
   name: auto-kube-dev
   version: master
+- name: redhat-nfvpe.vm-spinup
+  version: master
```

Comment:
we could also tag a new version and pin it here? I'm ok with master for now though.

Reply:
sounds good, maybe worth pinning in the future if one leap frogs